### DISCUSSION TITLE
Understanding data gaps and associated files

### DISCUSSION THREAD

**Discussion by jfmartinez4** [OvertureMaps/data#221] [Discussion]
Hello,  I am having a hard time understanding what areas of the map are represented by which files from AWS, or how to download the correct file for the area I'm missing. There's a large area around Vallejo and Fairfield California, which I cannot identify which parquet file to download from AWS. Thank you. 

### Category Feedback

Add details 
 I downladed the Buildings parquet files and wanted to understand how the buildings were partitioned into the files. Whether it's by latitude and longitude tiles, or if it's another schema.

_Originally posted by @jfmartinez4 in https://github.com/OvertureMaps/data/discussions/195_

**Comment by jfmartinez4** [comment]
Here is an image:  
<img width="662" alt="Screenshot 2024-09-01 at 1 53 23 PM" src="https://github.com/user-attachments/assets/c88b486b-8e77-467a-8d30-ff910f694d2a">

**Comment by jwass** [comment]
Hi @jfmartinez4 - If you just want some data in an area, we recommend using either the Python download tool (https://github.com/OvertureMaps/overturemaps-py/) or the explore site (https://explore.overturemaps.org).

There are ways to find the exact files to download that cover an area of interest, but if the above tools work I'd recommend going that way.
